### PR TITLE
DocumentDB quickstart templates

### PR DESCRIPTION
**Pull request by ryancrawcour**
added two DocumentDB quickstart templates
* 101-create-documentdb-account
* 101-create-documentdb-account-ext

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@ryancrawcour__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ryancraw). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by singhkay** [comment]
@template-deploy-bot test this please

File: 101-create-documentdb-account-ext/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    "offerType": "Standard"
+  },
+  "resources": [
+    {
+      "apiVersion": "2015-04-08",
+      "type": "Microsoft.DocumentDB/databaseAccounts",
+      "name": "[parameters('databaseAccountName')]",
+      "location": "[resourceGroup().location]",
+      "properties": {
+        "name": "[parameters('databaseAccountName')]",
```
Comment (singhkay) [inline 101-create-documentdb-account-ext/azuredeploy.json]: This seems redundant with the name above as well on line 50

File: 101-create-documentdb-account-ext/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "https://ryancrawcour.blob.core.windows.net/schemas/2015-01-01/deploymentTemplate.json#",
```
Comment (singhkay) [inline 101-create-documentdb-account-ext/azuredeploy.json]: This needs to be azure deployment schema

File: 101-create-documentdb-account/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://ryancrawcour.blob.core.windows.net/schemas/2015-01-01/deploymentTemplate.json#",
```
Comment (singhkay) [inline 101-create-documentdb-account/azuredeploy.json]: This needs to be azure deployment schema

File: 101-create-documentdb-account-ext/README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Create DocumentDB Account with a specified Consistency Policy
```
Comment (singhkay) [inline 101-create-documentdb-account-ext/README.md]: What does ext mean in the template folder name?

Comment (ryancrawcour) [inline 101-create-documentdb-account-ext/README.md]: That's what the current RP (and schema) requires.

File: 101-create-documentdb-account-ext/azuredeploy.json (hunk 1 above)
Comment (ryancrawcour) [inline 101-create-documentdb-account-ext/azuredeploy.json]: This is what the RP wants currently.

File: 101-create-documentdb-account/azuredeploy.json (hunk 3 above)
Comment (ryancrawcour) [inline 101-create-documentdb-account/azuredeploy.json]: Good catch. Will have to fix that.

File: 101-create-documentdb-account-ext/azuredeploy.json (hunk 2 above)
Comment (ryancrawcour) [inline 101-create-documentdb-account-ext/azuredeploy.json]: Good catch. Will fix.

File: 101-create-documentdb-account-ext/README.md (hunk 4 above)
Comment (ryancrawcour) [inline 101-create-documentdb-account-ext/README.md]: ext means its "extended" or not basic, which the other one is. This one adds in consistency policy. Can rename this if you like.

Comment (singhkay) [inline 101-create-documentdb-account-ext/README.md]: Yep, please do to 'consistent-policy' instead of 'ext'

Comment (ryancrawcour) [inline 101-create-documentdb-account-ext/README.md]: done

File: 101-create-documentdb-account/azuredeploy.json (hunk 3 above)
Comment (ryancrawcour) [inline 101-create-documentdb-account/azuredeploy.json]: done

File: 101-create-documentdb-account-ext/azuredeploy.json (hunk 2 above)
Comment (ryancrawcour) [inline 101-create-documentdb-account-ext/azuredeploy.json]: done

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**